schema/filter_artworks.js

```diff
@@ -24,6 +28,20 @@ export const FilterArtworksType = new GraphQLObjectType({
       description: 'Artwork results.',
       type: new GraphQLList(Artwork.type),
     },
+    artworks_connection: {
+      type: artworkConnection,
+      args: pageable(),
+      resolve: ({ hits, aggregations }, options) => {
+        if ((!aggregations && !aggregations.total)) {
+          throw new Error('This query must contain the total aggregation');
```

Comment:
The `total` field isn't available unless you specifically request the aggregation. For instance if you query ` counts { total } ` without specifying the `total` aggregation you will get an error also.